src/main/webapp/i18n/en/assessmentDashboard.json

```
@@ -30,10 +30,10 @@
             "assessmentsDueDate": "Assessment due date",
             "actions": "Actions",
             "notReleased": "Not released",
-            "readGradingInstructions": "Read grading instructions",
-            "trainOnExampleSubmissions": "Practice with example submissions",
-            "assessStudentsSubmissions": "Assess students' submissions",
-            "assessStudentsLateSubmissions": "Assess students' late submissions",
+            "readGradingInstructions": "Read grading instructions and click on confirmation button below",
```

I am not sure how this is meant, it reads like a request, but is used as a status. Same goes for the other strings. I would expect it to be
```suggestion
            "readGradingInstructions": "Read grading instructions and clicked on confirmation button below",
```
but perhaps I missunderstand it